yt_dlp/extractor/porn91.py

```diff
@@ -29,32 +51,42 @@ def _real_extract(self, url):
         webpage = self._download_webpage(
             'http://91porn.com/view_video.php?viewkey=%s' % video_id, video_id)
 
-        if '作为游客，你每天只可观看10个视频' in webpage:
-            raise ExtractorError('91 Porn says: Daily limit 10 videos exceeded', expected=True)
+        if '作为游客，你每天只可观看15个视频' in webpage:
+            raise ExtractorError('91 Porn says: Daily limit 15 videos exceeded', expected=True)
 
-        title = self._search_regex(
-            r'<div id="viewvideo-title">([^<]+)</div>', webpage, 'title')
-        title = title.replace('\n', '')
+        title = self._html_extract_title(webpage)
+        title = remove_end(title.replace('\n', ''), 'Chinese homemade video').strip()
 
         video_link_url = self._search_regex(
-            r'<textarea[^>]+id=["\']fm-video_link[^>]+>([^<]+)</textarea>',
-            webpage, 'video link')
-        videopage = self._download_webpage(video_link_url, video_id)
+            r'document\.write\(\s*strencode2\s*\(\s*((?:"[^"]+")|(?:\'[^\']+\'))\s*\)\s*\)', webpage, 'video link')
+        video_link_url = self._search_regex(
+            r'src=["\']([^"\']+)["\']', urllib.parse.unquote(video_link_url), 'unquoted video link')
 
-        info_dict = self._parse_html5_media_entries(url, videopage, video_id)[0]
+        upload_date = self._search_regex(
+            r'<span\s+class=["\']title-yakov["\']>(\d{4}-\d{2}-\d{2})</span>', webpage, 'upload_date', fatal=False)
+        upload_date = unified_strdate(upload_date)
+
+        description = self._html_search_regex(
+            r'<span\s+class=["\']more title["\']>\s*(.*(?!</span>))\s*</span>', webpage, 'description', fatal=False)
```

Comment:
The regex is wrong. You have to group the `.` with `(?!)` and `*` the whole thing to do what you want. But it's better to just do:
```suggestion
            r'<span\s+class=["\']more title["\']>\s*([^<]+', webpage, 'description', fatal=False)
```